Mesh_3/include/CGAL/Mesh_3/Refine_facets_3.h

```diff
@@ -483,7 +483,7 @@ class Refine_facets_3_base
   bool is_encroached_facet_refinable(Facet& facet) const;
 
   /// Insert facet into refinement queue
-  void insert_bad_facet(Facet facet, const Quality& quality)
+  void insert_bad_facet(Facet& facet, const Quality& quality)
```

Comment:
Thank you for asking!
The reverse modification was made by you in the commit c55e391f97eba43e93a2b9232b57f69c1a08174a, in the context of making the manifold criterion available with Mesh_3 parallel (PR #1911).
I must have mis-synchronized our private branch and this one. I'm fixing it!